pysat/__init__.py

```diff
@@ -84,9 +84,9 @@
     # load up stored user modules
     user_modules = []
     with open(os.path.join(pysat_dir, 'user_modules.txt'), 'r') as f:
-        _ = f.readline()
-        if _ != '' and (_ is not None):
-            user_modules.append(_)
+        for _ in f:
```

Comment:
I did start with the _ variable name but that was a poor choice on my part. perhaps _temp ?